client/client.go

```diff
@@ -451,10 +451,30 @@ func (c *Client) setupNode() error {
 	return nil
 }
 
+func (c *Client) compileOptionsList(key string) (map[string]struct{}, bool) {
+	s := strings.TrimSpace(c.config.ReadDefault(key, ""))
+	list := make(map[string]struct{})
+	if s != "" {
+		for _, driver := range strings.Split(s, ",") {
+			trimmed := strings.TrimSpace(driver)
+			list[trimmed] = struct{}{}
+		}
+	}
+	enabled := len(list) > 0
+	return list, enabled
+}
+
 // fingerprint is used to fingerprint the client and setup the node
 func (c *Client) fingerprint() error {
+	blacklist := c.config.ReadStringListToMap("fingerprint.blacklist")
```

Comment:
Can we actually make this a whitelist. In the future if we have fingerprinters from plugins, you may not know the full set of fingerprinters until runtime, so a whitelist works better there and then we wouldn't have to change the API down the line. It is a bit more verbose but will be easier to maintain.